package.json

```diff
@@ -59,6 +59,7 @@
     "@typescript-eslint/eslint-plugin": "^6.12.0",
     "@typescript-eslint/parser": "^6.12.0",
     "cypress": "^13.7.2",
+    "cypress-file-upload": "^5.0.8",
```

Comment:
This seems fine to me but the `item.cy.js` tests have been uploading an image without this extension, is there a reason why we needed to add this?

Reply:
Thanks for bringing this up, @camelPhonso! I initially installed this based on a YouTube tutorial I watched, but then I realised that this is for older versions of Cypress and implemented a code without it. 

I've removed the dependency with my new commit, as it is not being used by neither `userProfile.cy.js` nor ` item.cy.js`